src/main/java/uk/gov/hmcts/reform/iahomeofficeintegrationapi/domain/service/HomeOfficeInstructService.java

```diff
@@ -0,0 +1,77 @@
+package uk.gov.hmcts.reform.iahomeofficeintegrationapi.domain.service;
+
+import static uk.gov.hmcts.reform.iahomeofficeintegrationapi.infrastructure.config.HomeOfficeProperties.LookupReferenceData;
+
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.iahomeofficeintegrationapi.domain.entities.ConsumerInstruct;
+import uk.gov.hmcts.reform.iahomeofficeintegrationapi.domain.entities.ConsumerReference;
+import uk.gov.hmcts.reform.iahomeofficeintegrationapi.domain.entities.ConsumerType;
+import uk.gov.hmcts.reform.iahomeofficeintegrationapi.domain.entities.CourtOutcome;
+import uk.gov.hmcts.reform.iahomeofficeintegrationapi.domain.entities.CourtType;
+import uk.gov.hmcts.reform.iahomeofficeintegrationapi.domain.entities.HomeOfficeInstruct;
+import uk.gov.hmcts.reform.iahomeofficeintegrationapi.domain.entities.MessageHeader;
+import uk.gov.hmcts.reform.iahomeofficeintegrationapi.domain.entities.MessageType;
+import uk.gov.hmcts.reform.iahomeofficeintegrationapi.domain.entities.Outcome;
+import uk.gov.hmcts.reform.iahomeofficeintegrationapi.infrastructure.client.HomeOfficeInstructApi;
+import uk.gov.hmcts.reform.iahomeofficeintegrationapi.infrastructure.client.util.HomeOfficeApiUtil;
+import uk.gov.hmcts.reform.iahomeofficeintegrationapi.infrastructure.config.HomeOfficeProperties;
+
+@Service
+@Slf4j
+public class HomeOfficeInstructService {
+
+    private final HomeOfficeProperties homeOfficeProperties;
+    private final HomeOfficeInstructApi homeOfficeInstructApi;
+
+    public HomeOfficeInstructService(
+        HomeOfficeProperties homeOfficeProperties, HomeOfficeInstructApi homeOfficeInstructApi) {
+        this.homeOfficeProperties = homeOfficeProperties;
+        this.homeOfficeInstructApi = homeOfficeInstructApi;
+    }
+
+    public MessageHeader sendNotification(String homeOfficeReferenceNumber, String caseId) {
+
+        HomeOfficeInstruct request = makeRequestBody(homeOfficeReferenceNumber, caseId);
+        MessageHeader instructResponse = makeRequest(request);
+        log.debug("HomeOffice-Instruct response: {}", instructResponse);
+
+        return instructResponse;
+    }
+
+    public MessageHeader makeRequest(HomeOfficeInstruct request) {
+
+        return homeOfficeInstructApi.sendNotification(request);
```

Comment:
not sure there's benefit in having this in it's own method here.

Reply:
This is already fixed in #77

Reply:
Please fix this in this PR.

Reply:
It is fixed